### PR TITLE
virtualenvwrapper: add page

### PR DESCRIPTION
**Pull request by aitzaz**
- [ ] The page (if new), does not already exist in the repo.
- [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [ ] The page has 8 or fewer examples.
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by zdroid** [comment]
I edited this to fix some obvious errors, but I'm also leaving some review comments.

File: pages/common/virtualenvwrapper.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# virtualenvwrapper
+
+> Group of simple wrapper commands for Python's `virtualenv` tool. 
+> More information: <http://virtualenvwrapper.readthedocs.org>.
+
+- Create a new Python `virtualenv`; It will be created in `$WORKON_HOME`:
```
Comment (zdroid) [inline pages/common/virtualenvwrapper.md:6]:
```suggestion
- Create a new Python `virtualenv` in `$WORKON_HOME`:
```

File: pages/common/virtualenvwrapper.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+`mkvirtualenv {{virtual_env_name}}`
```
Comment (zdroid) [inline pages/common/virtualenvwrapper.md:8]:
```suggestion
`mkvirtualenv {{virtualenv_name}}`
```

to be consistent, in other lines too

File: pages/common/virtualenvwrapper.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+> Group of simple wrapper commands for Python's `virtualenv` tool.
+> More information: <http://virtualenvwrapper.readthedocs.org>.
+
+- Create a new Python `virtualenv` in `$WORKON_HOME`:
+
+`mkvirtualenv {{virtualenv_name}}`
+
+- Create a `virtualenv` for a specific Python version:
+
+`mkvirtualenv --python {{/usr/local/bin/python2.7}} {{virtualenv_name}}`
```
Comment (sbrl) [inline pages/common/virtualenvwrapper.md:12]: Perhaps we could use `python3` as an example here? Python2 was discontinued back in January.

**Comment by aitzaz** [comment]
Thanks for the suggestion @sbrl ! I've updated PR.

**Pull request merged**